pkg/config/loader.go

```diff
@@ -136,7 +128,7 @@ func (l *Loader) mergeFile(file string) error {
 	return nil
 }
 
-func (l *Loader) mergeAWX(decoded *data.AWXConfig) error {
```

Comment:
`data.AWXConfig` is already a pointer, do we really need a pointer to a pointer?

Reply:
Sorry, that ^ was incorrect, please disregard.